### DISCUSSION TITLE
Error when add items to checkout Object

### DISCUSSION THREAD

**Discussion by CharmingDev1219** [community/community#35605] [Actions]
Hello
When I add some items to checkout, I am getting this error.
What should I do else for me to fix this error?
If you have any experience, kindly help me.
Thanks.
<img width="493" alt="46038952-514caa00-c0c1-11e8-8556-4879ff75935c" src="https://user-images.githubusercontent.com/88336861/194762983-ddb4af6a-6d22-4222-803a-ba516e14a747.png">

**Comment by tccpro** [comment]
Please use `Shopify-buy` object.
It will does work, I have experience and please refer my past project one.

https://github.com/HOT-DEV/Shopify-Mobile-APP-Karma-V.0.8.2
